ipaclient/remote_plugins/schema.py

```diff
@@ -522,7 +522,7 @@ def _write_schema(self, fingerprint):
 
     def _read(self, path):
         with zipfile.ZipFile(self._file, 'r') as zf:
-            return json.loads(zf.read(path).decode('utf-8'))
+            return json.loads(decode_json(zf.read(path)))
```

Comment:
bytes doesn't say anything about encoding, but AFAIK it is part of IPA so we can assume it is always  'utf-8' encoded